src/controls/controls-system.js

```diff
@@ -89,6 +85,12 @@ Crafty.s("Controls", {
         // internal object to store definitions
         this._dpads = {};
         this._triggers = {};
+
+        // listen to mouse events
+        Crafty.s("Mouse").bind("MouseDown", this.updateTriggers.bind(this));
+        Crafty.s("Mouse").bind("MouseUp", this.updateTriggers.bind(this));
+        Crafty.s("Mouse").bind("DoubleClick", this.updateTriggers.bind(this));
+        Crafty.s("Mouse").bind("Click", this.updateTriggers.bind(this));
```

Comment:
> Is the issue here that the Mouse system now does not trigger the events globally?

Exactly, if Mouse system would trigger the events globally, all mouse entities would receive it.